eventuals/scheduler.h

```diff
@@ -53,7 +53,7 @@ class Scheduler {
 
     Context(Context&& that) = delete;
 
-    virtual ~Context() {}
+    virtual ~Context() = default;
```

Comment:
Let's actually make `Context` be `final`!

Reply:
Let's chat about that synchronously.